its/projects/CSharpVBNetCoverage/CSharpConsoleApp.Test/CSharpConsoleApp.Test.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
For some reason, the build was failing saying that netcoreapp2.1 cannot reference a net461 project.